library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/ViewUtilsInternal.kt

```diff
@@ -12,7 +12,7 @@ import android.view.ViewStub
 import androidx.appcompat.widget.ActionBarContextView
 import androidx.appcompat.widget.Toolbar
 
-internal class ViewUtils {
+internal class ViewUtilsInternal {
```

Comment:
are we splitting into 2 classes just because one has to stay simple `class` and another `object class`? I guess making this class `public` with some public members makes no sense, because then we have to use `object class` to follow the common convention for utility classes?

Reply:
nope...is just because one is meant for internal usage only as those method will not be needed externally and it also has a state due to the `Build.Version` check.